Fixed the issue!!

HakuG opened the pull request:
fixes #578 
fixes #537 
fixes #632 
fixes #753 
fixes #555

Commit:
Trying to remove Source in Testimonial

Commit:
Trying to remove Source in Testimonial

Commit:
Trying to remove Source in Testimonial

Commit:
Trying to remove Source in Testimonial

Commit:
Trying to remove Source in Testimonial

Commit:
Trying to remove Source in Testimonial

Commit:
Trying to remove Source in Testimonial

Commit:
Trying to remove Source in Testimonial

Commit:
Trying to remove Source in Testimonial

Commit:
Trying to remove Source in Testimonial

Commit:
Trying to remove Source in Testimonial

Commit:
Trying to remove Source in Testimonial

Commit:
Trying to remove Source in Testimonial

Commit:
Trying to remove Source in Testimonial

Commit:
Trying to remove Source in Testimonial

Commit:
Changing the search bar thing

Commit:
Changing the search bar thing

Commit:
Changing the search bar thing

Commit:
Changing the search bar thing

Commit:
Changing the search bar thing

Commit:
Changing the search bar thing

Commit:
Changing the search bar thing

Commit:
Changing the search bar thing

Commit:
Changing the search bar thing

Commit:
Changing the search bar thing

Commit:
Working on Anonymous

Commit:
Working on Anonymous

Commit:
Working on Anonymous

Commit:
Working on Anonymous

Commit:
Working on Anonymous

Commit:
Working on Anonymous

Commit:
Working on Anonymous

Commit:
Working on Anonymous

Commit:
Working on Anonymous

Commit:
Working on overflow

Commit:
Working on overflow

Commit:
Working on overflow

Commit:
Working on mediacreatorchange

Commit:
Pretty much done with mediacreatorchange. Have to check for possible bugs

nickkapur commented:
@HakuG 

Hey Harnek, to reference the issues your pull request fixes, use the hash # instead of the @ sign. I've amended your pull request text accordingly, so please see above for the correct notation to use going forward. If you use @, it points to a github user, which is why we got a baffled comment. Also, I removed the reference to #743, an issue which you almost certainly did not fix!

HakuG commented:
Opps. I meant to fix #555. I think I mistakenly put 743 and 733, and
figured that since the number of problems I had fixed was right, it was
right. my bad! I changed it.

On Tue, Mar 18, 2014 at 12:02 AM, nickkapur notifications@github.comwrote:

> @HakuG https://github.com/HakuG
> 
> Hey Harnek, to reference the issues your pull request fixes, use the hash
> 
> # instead of the @ sign. I've amended your pull request text accordingly,
> 
> so please see above for the correct notation to use going forward. If you
> use @, it points to a github user, which is why we got a baffled comment.
> Also, I removed the reference to #743https://github.com/Japan-Digital-Archives/Japan-Digital-Archive/issues/743,
> an issue which you almost certainly did not fix!
> 
> ## 
> 
> Reply to this email directly or view it on GitHubhttps://github.com/Japan-Digital-Archives/Japan-Digital-Archive/pull/757#issuecomment-37897732
> .
